### PR TITLE
added brightness adjustmnet script and related bash script to call it

### PR DESCRIPTION
**Pull request by sudoursa**
The readme file was also updated with usage information on the new feature.

This closes "enhancement to dim the wallpaper #8"

### CODE REVIEW

File: setBrightness.sh [hunk 1]
```diff
@@ -0,0 +1,16 @@
+TYPE=$1
+LEVEL=$2
+
+echo "Setting wallpaper brightness..."
+
+WALLPAPER_PATH="$(pwd)/wallpaper.png"
+
+nice python adjustWallpaper.py $TYPE $LEVEL $WALLPAPER_PATH
```
Comment (shardul08) [inline setBrightness.sh]: Could you change `python` to `python3` as users with default 2.x version can run into error.

File: adjustWallpaper.py [hunk 2]
```diff
@@ -0,0 +1,34 @@
+import sys
+from PIL import Image, ImageEnhance
+
+argumentList = sys.argv
+print(argumentList)
```
Comment (shardul08) [inline adjustWallpaper.py]: Remove unnecessary print statements.

Comment (sudoursa) [reply adjustWallpaper.py]: updated the pull request

File: setBrightness.sh (hunk 1 above)
Comment (sudoursa) [reply setBrightness.sh]: updated the pull request

**Comment by shardul08** [comment]
Merged. Thank you!

**Pull request merged**